VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/tests/participant-waiting-room.component.eventhub.events.spec.ts

```diff
@@ -304,6 +306,7 @@ describe('ParticipantWaitingRoomComponent event hub events', () => {
         participantStatusSubject.next(message);
 
         expect(component.participant.status).toBe(message.status);
+        expect(component.displayDeviceChangeModal).toBeFalsy();
```

Comment:
is displayDeviceChangeModal set to true in the setup?

Reply:
no, this is just to cover the reality of the modal being open

Reply:
Updated additional test setups where this is expected to be false